m4/geany-doxygen.m4

```diff
@@ -7,7 +7,7 @@ AC_DEFUN([GEANY_CHECK_DOXYGEN],
 			[AS_HELP_STRING([--enable-api-docs],
 					[generate API documentation using Doxygen [default=no]])],
 			[geany_with_doxygen="$enableval"],
-			[geany_with_doxygen="auto"])
+			[geany_with_doxygen="no"])
```

Comment:
I think the bug is a couple lines up where it says `[default=no]`, it should probably say `[default=auto]`.

Reply:
Yes, auto should be the default.

Reply:
fixed